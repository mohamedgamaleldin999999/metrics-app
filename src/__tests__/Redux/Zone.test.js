import zoneReducer, { getZone } from '../../redux/Zone';

jest.mock('axios', () => []);
describe('Fetch Component', () => {
  it('should render Cities Component', () => {
    expect(zoneReducer({ zones: [] }, { type: '' })).toMatchObject({});
    expect(getZone()).toMatchSnapshot();
  });
});
