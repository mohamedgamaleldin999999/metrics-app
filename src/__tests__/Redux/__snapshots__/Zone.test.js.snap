// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fetch Component should render Cities Component 1`] = `[Function]`;
